app/controllers/api/v1/my_modules_controller.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class MyModulesController < BaseController
+      before_action :load_team
+      before_action :load_project
+      before_action :load_experiment
+      before_action :load_task, only: :show
+      before_action :load_task_relative, only: %i(outputs output inputs input)
+
+      def index
+        tasks = @experiment.my_modules
+                           .page(params.dig(:page, :number))
+                           .per(params.dig(:page, :size))
+
+        render jsonapi: tasks, each_serializer: MyModuleSerializer
+      end
+
+      def show
+        render jsonapi: @my_module, serializer: MyModuleSerializer
+      end
+
+      def outputs
+        outputs = @my_module.my_modules
+                            .page(params.dig(:page, :number))
+                            .per(params.dig(:page, :size))
+        render jsonapi: outputs, each_serializer: MyModuleSerializer
+      end
+
+      def output
+        output = @my_module.my_modules.find(params.require(:id))
+        render jsonapi: {}, status: :not_found if output.nil?
```

Comment:
There is no need for this check, find at previous line will raise an exception if item not found